aiohue/v2/controllers/base.py

```diff
@@ -234,11 +234,10 @@ async def _handle_event(
             # in fact this is a feature request to Signify to handle these stateless
             # device events in a different way:
             # https://developers.meethue.com/forum/t/differentiate-stateless-events/6627
-            if self.item_type == ResourceTypes.BUTTON and not evt_data.get("button"):
+            if self.item_type == ResourceTypes.BUTTON and not evt_data.get("button", {}).get("button_report"):
```

Comment:
```suggestion
            if self.item_type == ResourceTypes.BUTTON and not evt_data.get("button_report"):
```

This can be simplified now that the old button is phased out. Can you test it with this change ?